examples/Boost-thread/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# download boost
+hunter_add_package(Boost COMPONENTS system thread)
```

Comment:
We are using `Boost::chrono` in fact, see foo.cpp. I guess that's the reason why test fail on Windows: https://ci.appveyor.com/project/ingenue/hunter/build/1.0.912/job/g631xocheg8xhsd9